rl/util.py

```diff
@@ -90,11 +91,19 @@ def __init__(self, optimizer, additional_updates):
         self.optimizer = optimizer
         self.additional_updates = additional_updates
 
-    def get_updates(self, params, constraints, loss):
-        updates = self.optimizer.get_updates(params, constraints, loss)
-        updates += self.additional_updates
-        self.updates = updates
-        return self.updates
+    if int(keras_version.split('.')[0]) == 1:
```

Comment:
It may be better to just try to call the Keras 2 version, catch the `TypeError` exception and then fall back to the Keras 1 behavior.